tests/test_config.py

```diff
@@ -0,0 +1,34 @@
+from ..src import config as cf
+import numpy as np
+
+
```

Comment:
What happened to the test functions testing whether paths exist (the ones we covered yesterday)?

Reply:
So those test functions wouldn't when pushing to github because the path doesn't actually exist on github. Those paths are data paths which are in my .gitignore file.